template/runtime/Cargo.toml

```diff
@@ -23,6 +23,7 @@ system = { version = "2.0.0-dev", default-features = false, package = "frame-sys
 timestamp = { version = "2.0.0-dev", default-features = false, package = "pallet-timestamp", path = "../../vendor/substrate/frame/timestamp" }
 transaction-payment = { version = "2.0.0-dev", default-features = false, package = "pallet-transaction-payment", path = "../../vendor/substrate/frame/transaction-payment" }
 ethereum = { version = "0.1.0", default-features = false, package = "pallet-ethereum", path = "../../frame/ethereum" }
+ethereum-core = { version = "0.2", default-features = false, features = ["codec"], package = "ethereum"}
```

Comment:
Why do we need this dep? Doesn't `ethereum` crate already contain the `Block` struct?

Reply:
Yes, make little sense. I will add `pub use ethereum::Block` in `pallet-ethereum` instead, and use that in the Runtime.

Reply:
Ah really sorry I misunderstood this. This was a package rename, not a new rep. Either reexporting or keep it as it is seems all fine, up to you!